bw2data/utils.py

```diff
@@ -297,33 +296,30 @@ def download_file(filename, directory="downloads", url=None):
 
     assert isinstance(directory, str), "`directory` must be a string"
     dirpath = projects.request_directory(directory)
-    filepath = dirpath / filename
+    filepath = os.path.join(dirpath, filename)
```

Comment:
Please switch this back, we are using pathlib.Path instances throughout bw2data now. Thanks!